src/main/java/edu/greenblitz/robotName/commands/shooter/shootingState/GoToShootingStateAndShoot.java

```diff
@@ -0,0 +1,19 @@
+package edu.greenblitz.robotName.commands.shooter.shootingState;
+
+import edu.greenblitz.robotName.commands.shooter.Shoot;
+import edu.greenblitz.robotName.shootingStateService.ShootingState;
+import edu.greenblitz.robotName.utils.GBCommand;
+
+public class GoToShootingStateAndShoot extends GBCommand {
+    @Override
+    public void initialize() {
+        new GoToShootingState();
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        if(ShootingState.isRobotInShootingPosition()) {
+            new Shoot();
+        }
+    }
```

Comment:
this command does not work and even if you did .schedule() it wouldn't have worked cuz you need a sequential command of a conditional command